tests/test_roles.py

```diff
@@ -0,0 +1,27 @@
+import edgedb
+
+from edb.testbase import server as tb
+
+
+class TestDescribeRoles(tb.QueryTestCase):
+
+    async def test_describe_roles(self):
+        await self.con.execute("""
+            CREATE SUPERUSER ROLE base1;
+            CREATE SUPERUSER ROLE `base 2`;
+            CREATE SUPERUSER ROLE child1 EXTENDING base1;
+            CREATE SUPERUSER ROLE child2 EXTENDING `base 2`;
+            CREATE SUPERUSER ROLE child3 EXTENDING base1, child2 {
+                SET password := 'test'
+            };
+        """)
+        roles = next(iter(await self.con.fetchall("""DESCRIBE ROLES""")))
+        base1 = roles.index('CREATE SUPERUSER ROLE `base1`;')
+        base2 = roles.index('CREATE SUPERUSER ROLE `base2`;')
+        child1 = roles.index('CREATE SUPERUSER ROLE `child1`')
+        child2 = roles.index('CREATE SUPERUSER ROLE `child2`')
+        child3 = roles.index('CREATE SUPERUSER ROLE `child3`')
+        assert child1 > base1, roles
+        assert child2 > base2, roles
+        assert child3 > child2 and child3 > base1, roles
+        assert "SET password_hash := 'SCRAM-SHA-256$4096:" in roles, roles
```

Comment:
Need a similar test for `DESCRIBE CONFIG`

Reply:
As I've written in #1497, I don't know how to make a valid test for this. Should I create a new instance just for DESCRIBE CONFIG test? We can probably make an instance for `dump --all` and then reuse it for `DESCRIBE CONFIG`, because too many instances doesn't sound a great idea.

Reply:
Can you just run `DESCRIBE CONFIG` and check that the command was successful at all? That's a very surface test, but still a test.

Reply:
Ah, yes. At the point of writing this, I've assumed that syntax tests do run queries. But now as I know they aren't I've added some smoke test.